docs/wxm-devices/D1-bundle/D1-connect-and-claim.mdx

```diff
@@ -0,0 +1,202 @@
+---
+id: D1-connect-and-claim
+title: Connect and Claim Station
+sidebar_label: Connect and Claim Station
+slug: /wxm-devices/D1-bundle/D1-connect-and-claim
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+## WG1200 Gateway
+
+<img object-fit="contain" src={useBaseUrl("/img/home/wg1200-gateway.png")} />
+
+The weather station comes together with the WG1200 Gateway.
+The WG1200 Gateway receives all the meteorological data plus the location of the weather station, which are measured and transmitted every 16 seconds by the outdoor sensor
+and displays them on its screen.
+
+
+In this chapter, you will find guidance to set up the WG1200 Gateway.
+<hr/>
+
+## Prerequisites
+
+1. The station must be fully assembled (parts and batteries).
+1. A working smartphone with connection to the internet.
+3. A reliable 2.4 GHz WiFi network (**5 GHz is not supported**) for the WG1200 Gateway to connect to.
+
+<hr/>
+
+
+
+## Step by Step Instructions
+
+
+
+### Connect WG1200 Gateway to power source and turn the gateway on
+
+
+  
+- The WG1200 Gateway must be connected to a power source at all times.
+
+
+:::caution
+If WG1200 Gateway turns off, you will not receive $WXM rewards, as you will not be able to share data with the WeatherXM network. **This will result in no $WXM rewards!**
+WG1200 does not have a battery, so it must be connected to a power source at all times.
+:::
+
+
+<hr/>
+
+
+
+
+### Change Frequency (Optional)
+
+
+The D1 Gateway's frequency must be set to match the frequency of your weather station so it can listen for the weather data it transmits. There are 2 frequencies available:
+- Model A: 868 MHz (default)
+- Model B: 915 MHz
+
+:::note
+**If your weather station is 868 MHz, you don't need to change anything as it is configured to this value by default.**
+If unsure, check the sticker under the station or the box that came in it.
+:::
+
+
+<hr/>
+
+### Connect WG1200 Gateway to WiFi
+
+<div style={{ display: 'flex', alignItems: 'center', marginBottom: '20px' }}>
+  <img 
+    src={useBaseUrl("/img/wxm-devices/D1-bundle/D1-Wifi.jpg")} 
+    style={{ width: '250px', height: 'auto', objectFit: 'contain', marginRight: '20px' }} 
+  />
+  <div>
+    <p>By now, your WG1200 Gateway is set and ready for claiming! Swipe left to navigate to the settings screen. There, tap on "Configure WiFi" to switch to connect mode.</p>
+    <p>Once this is done, connect to WG1200 Gateway via WiFi (you should see in the networks screen a "WXM_WG1200").</p>
+    <p>A popup will open, asking you to select the desired network you'd like to connect the D1 Gateway. Select the one you want and provide the password for it.</p>
+    <p>In case of a successful connection, and the popup doesn’t open, navigate with your browser to 10.10.0.1 and follow the instructions displayed.</p>
```

Comment:
You don't need to add `<p></p>` to get new lines, you can just do this:

```
    By now, your WG1200 Gateway is set and ready for claiming! Swipe left to navigate to the settings screen. There, tap on "Configure WiFi" to switch to connect mode.

    Once this is done, connect to WG1200 Gateway via WiFi (you should see in the networks screen a "WXM_WG1200").

    A popup will open, asking you to select the desired network you'd like to connect the D1 Gateway. Select the one you want and provide the password for it.

    In case of a successful connection, and the popup doesn’t open, navigate with your browser to 10.10.0.1 and follow the instructions displayed.
```